DOCS-26 - Metrics DPM - GA (#3058)
SumoLogic/sumologic-documentation: blog-service/2023-10-27-metrics.md

---
title: Metrics DPM (Metrics)
image: https://www.sumologic.com/img/logo.svg
keywords:
  - metrics
  - volume
  - dpm
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

We're excited to announce the release of Metrics DPM, a new screen that provides detailed metrics data points per minute (DPM) information.

> Metrics DPM answers the question “What things are sending metrics data, and how much are they sending?” Metrics DPM not only provides high level information about the metrics volume (DPM), but also breaks it down by individual metric names, as well as specific dimensions like cluster, host, and source. With this information, you can determine the items that contribute most to your credit consumption.  

Reviewer: ```suggestion
Metrics DPM answers the question, “What things are sending metrics data, and how much are they sending?”. Metrics DPM not only provides high level information about the metrics volume (DPM), you'll also see a breakdown by individual metric name, as well as specific dimensions like cluster, host, and source. With this information, you can determine the items that contribute most to your credit consumption.
```
